### PR TITLE
issues 49

### PR DESCRIPTION
**Pull request by DarkNebula0**
Fix wallet data and small error when no identity is set

### CODE REVIEW

**Comment by 2075** [comment]
Okay, maybe for clarification, the expected behaviour:

1. when you enter the site for the first time there should not be an approval request coming from the polkadot-js wallet

2. if you enter /app route, wallet component should show "connect"

3. when you click it the first time, approval window will show, if not first time, it simply connects

currently the same bug, which i have here in my branch, also:

image

where it does not have any accounts at all after clicky connect.

**Comment by 2075** [comment]
when disconnected, this should not show ( see console ): image

File: src/components/AccountSelector.tsx [hunk 1]
```diff
@@ -1,63 +1,39 @@
...
-	const { keyring } = useSubstrate()
+const AccountComponent = () => {
+	const { keyring, loadAccounts, logout } = useSubstrate()
 	const { allowConnect, setAllowConnect, setAccountAddress, setAccountPair } = useWallet()
-	const [ keyringOptions, setKeyringOptions ] = useState(null)
-	const [ initialAddress, setInitialAddress ] = useState(null)
-	const [ accountSelected, setAccountSelected ] = useState(null)
-	const [ selectedIndex, setSelectedIndex ] = useState(0)
+	const [keyringOptions, setKeyringOptions] = useState(null)
+	// TODO: @2075 For what do we need the initialAddress I don't see any usage
```
Comment (DarkNebula0) [inline src/components/AccountSelector.tsx:20]: @2075

**Comment by DarkNebula0** [comment]
@2075 Do we already have some kind of notifications? I have found nothing in the code. 

If we don't have anything yet I would recommend this lib, very customizable and easy to use
https://github.com/fkhadra/react-toastify

**Comment by iZeroIO** [comment]
tested it on https://gamedao-beta-byurojw7g-75.vercel.app/app works as requested. Thanks.

**Pull request merged**